web/auth/oauth.go

```diff
@@ -238,9 +239,57 @@ func (a *AuthorizeHTTPHandler) authorizeForm(c echo.Context) error {
 		return c.Redirect(http.StatusFound, u.String()+"#")
 	}
 
+	if !params.client.Flagship {
+		flags, err := feature.GetFlags(inst)
+		if err != nil {
+			return err
+		}
+
+		if clientsLimit, ok := flags.M["cozy.oauthclients.max"].(float64); ok && clientsLimit >= 0 {
+			limit := int(clientsLimit)
+
+			clients, _, err := oauth.GetConnectedUserClients(inst, 100, "")
+			if err != nil {
+				return fmt.Errorf("Could not get user OAuth clients: %w", err)
+			}
+			count := len(clients)
+
+			if count >= limit {
```

Comment:
From what I understand of the documentation, it should be `count > limit`

Reply:
I'm confused. We are on the `authorize` page, the new OAuth client should have been created before we display it.

And the added part to the documentation says:

> In case a limit has been set on the Cozy to the number of user-connected OAuth
clients, and this limit has been exceeded already, the user will be presented a
screen requesting to either remove some existing clients or, if enabled,
increase the limit (e.g. by subscribing to a plan with a greater limit). Once
the number of connected clients is brought back to the limit or under, the
OAuth flow will resume and the permissions screen will be displayed.